### ISSUE TITLE
Version `.env.example` as `.env`?

### ISSUE THREAD

**Issue by brianmcmichael** [OSC/ood-fileexplorer#146]
There's been some discussion and disagreement over whether to include the envvars as a requirement or to leave it as an example. I'm putting this out as an issue for discussion.

If it stays an example, the edit and terminal buttons won't work until the appropriate envvars are set.

If it is versioned to the default installation, the editor and terminal will point to our recommended system install locations. It is now assumed that `shell` and `file-editor` will be included as part of the base install. This also allows us to remove the copy step from the installation guide. The drawback is that if the `file-editor` and `shell` are not installed, or installed to a non-standard path, the buttons won't work.

**Comment by ericfranz** [comment]
In app.js we have this already:

```
        upload_max:             process.env.FILE_UPLOAD_MAX || 2097152000,
        file_editor:            process.env.OOD_FILE_EDITOR,
        shell:                  process.env.OOD_SHELL,
```

Lets change it to

```
        upload_max:             process.env.FILE_UPLOAD_MAX || 10485760000,
        file_editor:            process.env.OOD_FILE_EDITOR || "/pun/sys/file-editor/edit",
        shell:                  process.env.OOD_SHELL || "/pun/sys/shell/ssh/default",
```

I don't remember the reason we chose `2097152000` as the default file upload max but the example env has `10485760000` set.

Then we don't need a .env.example copy step. We could add a link to a wiki page about changing the file upload or disabling file_editor or shell links by adding a .env and then doing something like this:

```
OOD_FILE_EDITOR=""
```

But that probably won't happen.

**Comment by brianmcmichael** [comment]
Closed by #147
